README.md

```diff
@@ -117,6 +117,62 @@ cron.schedule('* * * Jan,Sep Sun', function(){
 });
 ```
 
+## Cron methods
+
+### Schedule
+
+Schedules given task to be executed whenever the cron expression ticks.
+
+Arguments:
+
+- !string expression - Cron expression
+- !Function func - Task to be executed
+- boolean? immediateStart - Whether to start scheduler immediately after create.
```

Comment:
I think a `ScheduledTask` must be running after creation and create without starting is the exception...